src/EPR.Payment.Service.UnitTests/Services/ResubmissionFees/ComplianceScheme/ComplianceSchemeResubmissionServiceTests.cs

```diff
@@ -29,23 +30,40 @@ public void Constructor_WhenResubmissionFeeStrategyIsNull_ShouldThrowArgumentNul
         {
             // Arrange
             IComplianceSchemeResubmissionStrategy<ComplianceSchemeResubmissionFeeRequestDto, decimal>? nullStrategy = null;
+            var paymentsServiceMock = _fixture.Create<Mock<IPaymentsService>>();
```

Comment:
do not create a fixture object. Use automoq to create the fixtures in test method. like;

![image](https://github.com/user-attachments/assets/cfd0b5c8-0ab6-400f-9032-33033466802c)